doc/dev/general.md

```diff
@@ -138,18 +138,22 @@ This has to happen before deploying kubecf.
 helm install stable/nginx-ingress \
   --name ingress \
   --namespace ingress \
-  --set "tcp.2222=kubecf/kubecf-scheduler:2222" \
```

Comment:
Please confirm the format , bz whole things will create through this internally .
It could be like this --set tcp.2222="kubecf/kubecf-scheduler:2222" \  
                 or
--set tcp:2222="kubecf/kubecf-scheduler:2222" \

I am still not sure ...

Reply:
it should be ok as it is with `--set tcp.2222="kubecf/kubecf-scheduler:2222"`, but more correct without `"`, like  `--set tcp.2222=kubecf/kubecf-scheduler:2222`.

As I see it, doing `--set "tcp.2222=kubecf/kubecf-scheduler:2222"` would mean that the `"` get expanded by the shell. Which in this case doesn't matter, anyways.

See https://helm.sh/docs/helm/helm_install/

Reply:
**Correct syntax  for eirini** : 
  --set tcp.2222="kubecf/eirinix-ssh-proxy:2222" 
**Correct Syntax for Diego**
  --set tcp.2222="kubecf/scheduler:2222" 

Is this right ?